nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2315,7 +2419,7 @@ kadmin: xst -k nifi.keytab nifi@EXAMPLE.COM
 
 This keytab file can be copied to the other NiFi nodes with embedded zookeeper servers.
 
-This will create a file in the current directory named `nifi.keytab`. We can now copy that file into the _$NIFI_HOME/conf/_ directory. We should ensure
+This will create a file in the current directory named `nifi.keytab`. We can now copy that file into the `$NIFI_HOME/conf/` directory. We should ensure
```

Comment:
Looks like in most other places, _filenames_ are denoted by italics, rather than `code formatting`.

Reply:
I've been going with this convention:  filenames in italics, directory paths/directories in code formatting.

Reply:
Cool, as long as it is consistent.